genie-test-web/src/main/resources/application.yml

```diff
@@ -29,7 +29,7 @@ cloud:
 genie:
   file:
     cache:
-      location: file:///tmp/genie/cache
+      location: file://${java.io.tmpdir}genie/cache
```

Comment:
Are you assuming that the java.io.tmpdir will not contain the scheme?

Reply:
In every example and test i've run it doesn't but yeah it's an assumption.